MANIFEST.in

```diff
@@ -1,3 +1,4 @@
 include LICENSE
 
 graft websockets/py35
+include websockets/cext.c
```

Comment:
Also https://docs.python.org/3/distutils/sourcedist.html#manifest suggests this isn't necessary?